contracts/oracle/FastPriceFeed.sol

```diff
@@ -49,6 +61,9 @@ contract FastPriceFeed is ISecondaryPriceFeed, IFastPriceFeed, Governable {
     mapping (address => bool) public isUpdater;
 
     mapping (address => uint256) public prices;
+    // store last Chainlink price, cumulative Chainlink price delta, cumulative fast price delta
```

Comment:
afaik compiler can do it for you? if you use custom structure like `struct Price { a uint160; b uint32; c uint32; }`  you won't need to manually retrieve value using bit operations
just fyi

Reply:
oh right, will try that